webapp/channels/src/components/widgets/users/avatar/avatar.tsx

```diff
@@ -50,13 +50,13 @@ type Attrs = HTMLAttributes<HTMLElement>;
 const isURLForUser = (url: string) => url.startsWith(Client4.getUsersRoute());
 const replaceURLWithDefaultImageURL = (url: string) => url.replace(/\?_=(\w+)/, '/default');
 
-const Avatar = ({
+const Avatar = forwardRef<HTMLImageElement | HTMLDivElement, Props & Attrs>(({
```

Comment:
I wonder if there is any "HTMLElement" type more generic that we can use here, instead of doing the castings.

Reply:
yes changed to HTMLElement